webpack/assets/javascripts/react_app/components/common/Dropdown.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Dropdown = ({ links, id }) => {
+  const listLinks = links.map((link, i) => {
+    const liKey = `notification-link-${i}-${id}`;
+
+    return (
+      <li key={liKey} id={liKey}>
+        <a target="_blank" href={link.href}>{link.title}</a>
+      </li>
+    );
+  });
+
+  return (
+    <div className="dropdown pull-right dropdown-kebab-pf">
+      <button className="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight{id}"
+               data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        <span className="fa fa-ellipsis-v"></span>
+      </button>
```

Comment:
Patternfly itself doesn't implement a javascript framework so it makes sense that it uses the bootstrap jquery plugins.

With Angular and React the paradigm moves away from direct manipulation markup and adopts the approach of rendering to reflect state. With React the concept of one way data flow is also introduced.

If you have a look at [AngularUI Bootstrap](https://angular-ui.github.io/bootstrap/) (the port of Bootstrap to Angular) you can see that it contains native AngularJS directives and has no dependency on either jQuery or Bootstrap's JS. I don't know why Angular Patternfly chose to not to implement in the same manner.

I think that our components will be easier to reason about if we adopt the pure React approach instead of mixing Reach with the Bootstrap plugins. Specifically, the bootstrap approach was not workable with the Navigation Drawer and Alert/Toasts (which you can see when I update the PR).

I think would be better to encapsulate a click handler in the component rather than rely on the boostrap plugins. In that way it will be easier for us to manage state, render in the React way and initiate actions.

@matanwerbner what do you think?